app/views/layouts/_header.html.haml

```diff
@@ -7,6 +7,7 @@
   - if user_signed_in?
     %li{class: active_class(my_profile_path), role: 'presentation'}= link_to "#{current_user.first_name} #{current_user.last_name}", my_profile_path
     %li{class: active_class(questions_path), role: 'presentation'}= link_to 'Questions', questions_path
+    %li{class: active_class(admin_daily_questions_path), role: 'presentation'}= link_to 'Liste quotidienne des questions', admin_daily_questions_path
```

Comment:
Maybe you could wait for the #135 to be merged, and then also add i18n for this partial?